<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DogsList</class>
 <widget class="QMainWindow" name="DogsList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1366</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dogs Shelter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../resources/images/logo-icon.png</normaloff>../resources/images/logo-icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #F8F3E0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">border-radius: 20px;</string>
   </property>
   <widget class="QFrame" name="navigationFrame">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>34</y>
      <width>1158</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #E9E4D4;
border-radius: 20px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="navigationMenuLogo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>18</y>
       <width>150</width>
       <height>45</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #017F36; 
height: 120px;
font-weight:  600;
font-family: Montserrat;
font-size: 20px;
color: #fff;</string>
     </property>
     <property name="text">
      <string>Doniraj</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>20</y>
       <width>341</width>
       <height>37</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>7</number>
      </property>
      <item>
       <widget class="QLabel" name="homeLabelButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #017F36;
font-size: 22px;
font-family: Montserrat;</string>
        </property>
        <property name="text">
         <string>Početna</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aboutUsLabelButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #017F36;
font-size: 22px;
font-family: Montserrat;</string>
        </property>
        <property name="text">
         <string>O nama</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="contactUsLabelButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #017F36;
font-size: 22px;
font-family: Montserrat;</string>
        </property>
        <property name="text">
         <string>Kontakt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="pawIconOnTheEdge_2">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>680</y>
      <width>132</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">opacity: 20%;
background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="exitButton">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>20</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dogsListHeaderTitle">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>160</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-weight:bold;
font-family: Montserrat;
font-size: 32px;</string>
    </property>
    <property name="text">
     <string>Dostupni psi</string>
    </property>
   </widget>
   <widget class="QLabel" name="dogsListHeaderDescription">
    <property name="geometry">
     <rect>
      <x>67</x>
      <y>194</y>
      <width>521</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #444;
font-weight:300;
font-family: Montserrat;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>Ovdje možete pronaći listu svih pasa spremnih za udomljavanje.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportButton">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>170</y>
      <width>150</width>
      <height>45</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #017F36; 
height: 120px;
font-weight:  bold;
font-family: Montserrat;
font-size: 20px;
color: #fff;</string>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>1241</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QFrame" name="dogOneFrame">
       <property name="minimumSize">
        <size>
         <width>291</width>
         <height>281</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>291</width>
         <height>281</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #D9D9D9;
border-radius: 20px;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>210</y>
          <width>291</width>
          <height>72</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(233,228,212, 0.9); 
border-top-left-radius: 0px !important;
border-top-right-radius: 0px !important;
border-bottom-left-radius: 20px !Important; 
border-bottom-right-radius: 20px !Important; </string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="dogsAge_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>40</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 14px;
color: #444; 
background-color: transparent;</string>
         </property>
         <property name="text">
          <string>Obučen, voli da se igra</string>
         </property>
        </widget>
        <widget class="QLabel" name="moreInfoIcon_4">
         <property name="geometry">
          <rect>
           <x>235</x>
           <y>11</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget_4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>10</y>
           <width>165</width>
           <height>32</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="dogsName_4">
            <property name="styleSheet">
             <string notr="true">font-size: 22px;
color: #017F36;
font-weight: bold;
background-color: transparent;</string>
            </property>
            <property name="text">
             <string>Pujdo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dogsAge_8">
            <property name="styleSheet">
             <string notr="true">font-size: 20px;
color: #444; 
background-color: transparent;</string>
            </property>
            <property name="text">
             <string>, 8 mjeseci</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QLabel" name="dogImage_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>251</width>
          <height>281</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: transparent;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
       <zorder>dogImage_4</zorder>
       <zorder>frame_5</zorder>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
